.gitignore

```diff
@@ -103,9 +103,6 @@ dist
 # TernJS port file
 .tern-port
 
-# webpack build ouputs
-.dist
```

Comment:
I think we need to get rid of the `dist`.

Reply:
It's already applied to PR.
Does it mean anything else?